master/getting-started/bare-metal/installation/index.md

```diff
@@ -5,13 +5,44 @@ canonical_url: 'https://docs.projectcalico.org/v3.2/getting-started/bare-metal/i
 
 You can use any of the following methods to install Felix on bare metal.
 
-- [Binary from package manager](binary-mgr): On Red Hat Enterprise Linux (RHEL), Ubuntu, 
+- [Binary from package manager](binary-mgr): On Red Hat Enterprise Linux (RHEL), Ubuntu,
   and CentOS hosts, use the package manager to install and run Felix as a binary.
-  
-- [Binary without package manager](binary): On CoreOS and Debian hosts, you can
-  install Felix without a package manager.
-  
+
 - [Container](container): On hosts equipped with Docker, you can install `{{site.nodecontainer}}`,
   which contains Felix and all of its dependencies.
 
+## Prerequisites
+
+Either way, you will need the following prerequisites.
+
+-  [Install calicoctl as a binary](/{{page.version}}/usage/calicoctl/install#installing-calicoctl-as-a-binary-on-a-single-host).
+
+-  [Configure calicoctl to connect to etcd](/{{page.version}}/usage/calicoctl/configure/).
+
+-  For IPv4 support, Linux kernel >=v2.6.32 is required.  The kernel's
```

Comment:
We actually have the requirements here....https://docs.projectcalico.org/master/getting-started/bare-metal/requirements

Reply:
Right, I will remove the duplicate material here.